examples/color/colormap_reference.py

```diff
@@ -67,7 +65,15 @@ def plot_color_gradients(cmap_category, cmap_list):
 for cmap_category, cmap_list in cmaps:
     plot_color_gradients(cmap_category, cmap_list)
 
-plt.show()
+
+###############################################################################
+# .. _reverse-cmap:
+# Reversing a colormap
+# --------------------
+#
+# A colormap can be reversed by appending ``_r`` to the name. For example:
```

Comment:
```suggestion
# The built in colormaps have a reversed version named with a ``_r`` 
# appended to the name. For example:
```

Adding the `_r` does not reverse the color map, we generate the reverse color maps by calling `cm.reversed()` in https://github.com/matplotlib/matplotlib/blob/5c4595267ccd3daf78f5fd05693b7ecbcd575c1e/lib/matplotlib/cm.py#L40-L57

It may be worth documenting the `reversed` method as well.